composer.json

```diff
@@ -60,6 +60,9 @@
             "drupal/entity": {
                 "Issue #3206703: Provide reverse relationships for bundle plugin entity_reference fields.": "https://www.drupal.org/files/issues/2022-05-11/3206703-10.patch"
             },
+            "drupal/entity_reference_revisions": {
```

Comment:
We should pin this dependency in composer.josn to its latest version. Otherwise our builds will start failing if/when they include this patch themselves.